classes/cluster/mk22_lab_basic/openstack/control.yml

```diff
@@ -31,6 +31,18 @@ parameters:
       package:
         python-msgpack:
           version: latest
+      repo:
+      # This repository need because python-influxdb package
+      # that is needed for Mitaka Ceilometer exists only
+      # in mos9.0-proposed suite for 9.0-latest
```

Comment:
Suggestion:
```
# This repository is needed because the python-influxdb package
# required for Mitaka Ceilometer is only present
# mos9.0-proposed from the 9.0-latest repository
```